backend/src/main/kotlin/cz/loono/backend/db/model/Badge.kt

```diff
@@ -0,0 +1,41 @@
+package cz.loono.backend.db.model
+
+import org.hibernate.envers.Audited
+import java.io.Serializable
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Embeddable
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.IdClass
+import javax.persistence.JoinColumn
+import javax.persistence.JoinColumns
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "\"badge\"")
```

Comment:
Nejaký super extra dôvod na dvojité "" ? Alebo chceme mať názov tabuľky malými písmenami?

Reply:
Uprime receno nevim, myslel jsem si, ze si PostgreSQL stezuje, kdyz nejsou doublequotes, ted jsem to vyzkousel a funguje to, jsem pouzival totiz stejny pristup jak mame pro ostatni model tridy, tak mozna @rposkocil vi o tom vice?

Reply:
V Oracle to funguje tak, že 
@Table(name = "badge") vytvorí tabuľku BADGE
@Table(name = "\"badge\"") vytvorí tabuľku badge
Myslím si, že v postgre je to opačne (https://stackoverflow.com/questions/43111996/why-postgresql-does-not-like-uppercase-table-names), takže zbytočné dávať "".

Reply:
Ok, tak dame to v ramci separatniho tasku na refactoring vsech entity? At neblokujeme toto PR

Reply:
Ok. Založíš prosím?

Reply:
Jo, #33